tools/csv-merger/csv-merger.go

```diff
@@ -333,6 +333,13 @@ func main() {
 
 			strategySpec := csvStruct.Spec.InstallStrategy.StrategySpec
 
+			// temporary workarounf for RHBZ#....
+			// a custom .spec.template.annotations["description"] it's causing a failure on OLM
+			// TODO: remove this once fixed on OLM side
+			for _, deployment := range strategySpec.DeploymentSpecs {
+				delete(deployment.Spec.Template.Annotations, "description")
```

Comment:
This handles a very specific annotation ("description").
Should we instead mimic the [OLM logic](https://github.com/operator-framework/operator-lifecycle-manager/blob/4b66803055a8ab611447c33ed86e755ad39cb313/pkg/controller/install/deployment.go#L263-L269) and just mirror the annotations from the CSV?

Reply:
we have more than one deployment in the CSV,
replicating the same long description more than once it's just going to make it unnecessarily long without any real value

Reply:
True, there's no value in replicating the same description over and over.